libs/navigation/navigation.css

```diff
@@ -36,4 +36,8 @@ header.global-navigation, header.global-navigation.feds--dark {
   .feds-promo-link:hover {
     color: #136FF6;
   }
+  .feds--full-width .feds-topnav {
```

Comment:
nit: we usually do a new line before a new declaration

Reply:
Done